tests/common/cached_transformers_test.py

```diff
@@ -7,12 +9,27 @@
 class TestCachedTransformers(AllenNlpTestCase):
     def test_get_missing_from_cache_local_files_only(self):
         with pytest.raises((OSError, ValueError)):
+            transformer = cached_transformers.get(
+                "bert-base-uncased",
+                True,
+                cache_dir=self.TEST_DIR,
+                local_files_only=True,
+            )
+            assert os.path.isfile(self.TEST_DIR + "/" + "bert-base-uncased.bin")
+            assert os.path.isfile(self.TEST_DIR + "/" + "config.json")
+            os.remove(self.TEST_DIR + "/" + "bert-base-uncased.bin")
+            os.remove(self.TEST_DIR + "/" + "config.json")
+
+            torch.save(transformer.module.state_dict(), self.TEST_DIR + "/" + "bert_weights.pth")
             cached_transformers.get(
                 "bert-base-uncased",
                 True,
+                override_weights_file=self.TEST_DIR + "/" + "bert_weights.pth",
                 cache_dir=self.TEST_DIR,
                 local_files_only=True,
             )
+            assert not os.path.isfile(self.TEST_DIR + "/" + "bert-base-uncased.bin")
+            assert os.path.isfile(self.TEST_DIR + "/" + "config.json")
```

Comment:
This is a good test, but you've put it inside an existing test method that is meant to test a different part of the functionality. I would be better to move the lines you've added to a new test method.